assembl/static2/css/views/idea.scss

```diff
@@ -30,6 +33,51 @@
     }
   }
 
+  .level-1,
+  .level-2,
+  .level-3,
+  .level-4 {
+    padding-bottom: $post-space;
+    padding-left: $tree-indentation*2;
+    border-top: 1px solid $first-color-light;
+    border-left: 1px solid $first-color-light;
+
+    .box {
+      border-top: none;
+    }
+
+    &:last-child {
+      border-left: none !important;
```

Comment:
is important necessary here ?

Reply:
I guess not, I was afraid to break something without noticing but it looks fine